pkg/jx/cmd/preview_test.go

```diff
@@ -141,8 +140,6 @@ preview:
 // functions).
 // TODO: Refactor the implementation & test so the various stages of creating a preview env. can be tested individually.
 func TestRun_CreateNewPreviewEnv(t *testing.T) {
-	t.Parallel()
```

Comment:
the code was using EnvironmentVariables which are global and inherently not suitable for parallelism - so removed this whilst I was here.